tests/unit/date/helper.js

```diff
@@ -0,0 +1,30 @@
+define( [
+	"jquery",
+	"globalize",
+	"lib/helper",
+	"globalize/date"
+], function( $, Globalize, helper ) {
+
+return $.extend( helper, {
+	getAttributes: function( locale ) {
```

Comment:
Or `$.ui.calendarDate` should reject a string as the first argument, only accepting a `Date`.